conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/DatabaseConversions.scala

```diff
@@ -418,6 +418,13 @@ object DatabaseConversions extends LazyLogging {
         )
         ) =>
       val (year, month, day, time) = extractDateTime(toSql(block.data.header.timestamp))
+      val paidStorageDiff = metadata.operation_result.paid_storage_size_diff.flatMap(extractBigDecimal)
+      val burn = paidStorageDiff.flatMap { psd =>
```

Comment:
I think we might need to document what's going on in this block.
I have problems figuring out what is actually computed for burn